foodsaving/users/templates/accountdelete_request-subject.jinja

```diff
@@ -1,3 +1 @@
-{% trans %}
 Request to delete your account
```

Comment:
Is still still translatable? I think you need some translation marker.

You can check by running `manage.py makemessages --ignore=env/* -l en` and see if the generated file `foodsaving/locale/en/LC_MESSAGES/django.po` contains this message.